creator-node/src/monitors/MonitoringQueue.js

```diff
@@ -37,33 +41,38 @@ class MonitoringQueue {
       }
     })
 
+    this.prometheusRegistry = prometheusRegistry
+
     // Clean up anything that might be still stuck in the queue on restart and run once instantly
     if (clusterUtils.isThisWorkerInit()) {
       this.queue.drain(true)
       this.seedInitialValues()
     }
-
-    const worker = new Worker(
-      'monitoring-queue',
-      async (job) => {
-        try {
-          await this.logStatus('Starting')
-
-          // Iterate over each monitor and set a new value if the cached
-          // value is not fresh.
-          Object.values(MONITORS).forEach(async (monitor) => {
-            try {
-              await this.refresh(monitor)
-            } catch (e) {
-              this.logStatus(`Error on ${monitor.name} ${e}`)
-            }
-          })
-        } catch (e) {
-          this.logStatus(`Error ${e}`)
-        }
-      },
-      { connection }
-    )
+    if (clusterUtils.isThisWorkerSpecial()) {
```

Comment:
could you add this same gating logic to `start()` in this class as well? I'd imagine otherwise each worker in the cluster would add jobs on an interval

Reply:
added